topics/ipfs/index.md

```diff
@@ -0,0 +1,19 @@
+---
+aliases: InterPlanetary-FileSystem, InterPlanetary FileSystem, Inter-Planetary FileSystem, IPFS, ipfs, InterPlanetary File System
```

Comment:
I checked on these and they don't look to be in use, or are the same as the topic `ipfs` itself. Can you remove this line? https://github.com/topics/interplanetary-file-system, https://github.com/topics/inter-planetary-filesystem, and https://github.com/topics/interplanetary-filesystem have no repositories.

If you want, you could add different aliases for topics that revolves around IPFS, where having info about IPFS would be useful to provide context, e.g.,

```yaml
aliases: ipfs-daemon, ipfs-api, ipfs-webui, js-ipfs, ipfs-protocol, go-ipfs
```

Reply:
Yeah, fixed this. My thinking was to avoid future topics to not be linked, but what you're saying is correct, those aliases are not used.